guides/auth-torii-with-github.md

```diff
@@ -0,0 +1,605 @@
+[Back to Main README](../README.md)
+
+# Full GitHub Authorization with the `torii-provider`
+
+This guide walks through the process of authenticating and authorizing an Ember app with
+and for use against GitHub using `ember-simple-auth` and `torii` including a registered
+GitHub application, an unauthenticated login route, an authenticated content route, a
+token exchange service, and use of an authorizer to inject the token into
+requests. We'll use the [ember-data-github](https://github.com/elwayman02/ember-data-github)
+addon so we don't need to create our own GitHub models, serializers, and adapters.
+
+While most of the pieces are documented in the `ember-simple-auth` and `torii`
+documentation, putting them together properly for the particular scenario has not been
+fully documented to date. Additionally, many of the sources on the general internet are
+helpful but dated or incomplete.
```

Comment:
I'm not sure it's necessary to explain the motivation for this guide here as with the release of this the problem that's being described doesn't exist anymore so the paragraph isn't actually relevant anymore ;)